packages/headless/src/features/question-answering/question-answering-slice.test.ts

```diff
@@ -48,4 +54,73 @@ describe('question answering slice', () => {
     expect(resulting.disliked).toBe(true);
     expect(resulting.liked).toBe(false);
   });
+
+  it('should handle expandSmartSnippetRelatedQuestion', () => {
+    state.relatedQuestions = [
+      {contentIdKey: 'foo', contentIdValue: 'bar', expanded: false},
+      {contentIdKey: 'foo', contentIdValue: 'bazz', expanded: false},
+    ];
+    const resulting = questionAnsweringReducer(
+      state,
+      expandSmartSnippetRelatedQuestion({
+        contentIdKey: 'foo',
+        contentIdValue: 'bazz',
+      })
+    );
+    expect(resulting.relatedQuestions[1].expanded).toBe(true);
+  });
+
+  it('should handle collapseSmartSnippetRelatedQuestion', () => {
+    state.relatedQuestions = [
+      {contentIdKey: 'foo', contentIdValue: 'bar', expanded: true},
+      {contentIdKey: 'foo', contentIdValue: 'bazz', expanded: true},
+    ];
+    const resulting = questionAnsweringReducer(
+      state,
+      collapseSmartSnippetRelatedQuestion({
+        contentIdKey: 'foo',
+        contentIdValue: 'bazz',
+      })
+    );
+    expect(resulting.relatedQuestions[1].expanded).toBe(false);
+  });
+
+  it('should handle executeSearch to populate relatedQuestions', () => {
+    const searchAction = executeSearch.fulfilled(
+      buildMockSearch({
+        response: buildMockSearchResponse({
+          questionAnswer: {
+            ...emptyQuestionAnswer(),
+            relatedQuestions: [
+              {
+                documentId: {contentIdKey: 'foo', contentIdValue: 'bar'},
+                score: 123,
+                answerSnippet: 'answer 1',
+                question: 'question 1',
+              },
+              {
+                documentId: {contentIdKey: 'buzz', contentIdValue: 'bazz'},
+                score: 321,
+                answerSnippet: 'answer 2',
+                question: 'question 2',
+              },
+            ],
+          },
+        }),
+      }),
+      '',
+      null as never
```

Comment:
?

Reply:
I found this pattern in other unit tests prior to this PR

Reply:
That's where the analytics payload would be passed in.

Not relevant for this UT, so just a `null`.